content/types/_client_options.textile

```diff
@@ -14,28 +14,29 @@ h4.
+- <span lang="default">tokenDetails</span><span lang="python">token_details</span><span lang="ruby">:token_details</span> := An authenticated "@TokenDetails@":/realtime/types/#token-details object that is most commonly obtained from of a token request response. Use this option if you wish to use Token authentication. Read more about "Token authentication":/general/authentication#token-authentication<br>__Type: @String@__
 
 - <span lang="default">tls</span><span lang="ruby">:tls</span> := _true_ A boolean value, indicating whether or not a TLS ("SSL") secure connection should be used. An insecure connection cannot be used with Basic authentication ensuring private keys are compromised in transit. "Find out more about TLS":https://support.ably.io/solution/articles/3000045208-are-messages-sent-to-and-received-from-ably-sent-security-using-tls<br>__Type: @Boolean@__
 
-- <span lang="default">clientId</span><span lang="ruby">:client_id</span> := A client ID, used for identifying this client when publishing messages or for presence purposes. The <span lang="default">@clientId@</span><span lang="ruby">@client_id@</span> can be any non-empty string. This option is primarily intended to be used in situations where the library is instanced with a key; note that a <span lang="default">@clientId@</span><span lang="ruby">@client_id@</span> may also be implicit in a token used to instance the library; an error will be raised if a <span lang="default">@clientId@</span><span lang="ruby">@client_id@</span> specified here conflicts with the <span lang="default">@clientId@</span><span lang="ruby">@client_id@</span> implicit in the token. "Find out more about client identities":/how-ably-works#client-identity<br>__Type: @String@__
+- <span lang="default">clientId</span><span lang="python">client_id</span><span lang="ruby">:client_id</span> := A client ID, used for identifying this client when publishing messages or for presence purposes. The <span lang="default">@clientId@</span><span lang="ruby">@client_id@</span> can be any non-empty string. This option is primarily intended to be used in situations where the library is instanced with a key; note that a <span lang="default">@clientId@</span><span lang="ruby">@client_id@</span> may also be implicit in a token used to instance the library; an error will be raised if a <span lang="default">@clientId@</span><span lang="ruby">@client_id@</span> specified here conflicts with the <span lang="default">@clientId@</span><span lang="ruby">@client_id@</span> implicit in the token. "Find out more about client identities":/how-ably-works#client-identity<br>__Type: @String@__
```

Comment:
missing python versions of clientId in the spans below, I expect you simply need to change `lang="ruby"` to `lang="ruby,python"` in most cases